### PR TITLE
Compile paraview v5.11.0

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@c-wetterer-nelson @patrickoleary Please review

**Review by burlen — CHANGES_REQUESTED** [review]
please use vtk version to select vtk features since those can be used without PV

File: sensei/CatalystAnalysisAdaptor.cxx [hunk 1]
```diff
@@ -28,7 +28,12 @@
 #include <vtkMultiProcessController.h>
 #include <vtkCPProcessor.h>
 #include <vtkPointSet.h>
+#if PARAVIEW_VERSION_MAJOR > 5 ||                                       \
+    (PARAVIEW_VERSION_MAJOR == 5 && PARAVIEW_VERSION_MINOR >= 11)
+#include <vtkPVVersion.h>
+#else
 #include <vtkPVConfig.h>
+#endif
```
[the remark this answers is not in the document]
Comment (danlipsa) [reply sensei/CatalystAnalysisAdaptor.cxx:36]: This is the full message of the deprecation:   "vtkPVConfig.h is deprecated. Please include `vtkPVVersion.h`, `pqQtConfig.h`, or `pqQtWidgetConfig.h` instead.")

[the remark this answers is not in the document]
Comment (danlipsa) [reply sensei/CatalystAnalysisAdaptor.cxx:36]: So it seems the original PVConfig had some QT stuff in it.

**Comment by danlipsa** [comment]
@burlen Any idea what is the issue with ecp-sdk-integration failing tests?

**Comment by burlen** [comment]
> @burlen Any idea what is the issue with ecp-sdk-integration failing tests?
All the tests fail like this:
```
mpiexec noticed that process rank 1 with PID 0 on node 277cde878462 exited on signal 4 (Illegal instruction).
```
I have no idea why that would be. @kwryankrattiger? @patrickoleary?

**Comment by danlipsa** [comment]
@burlen Lets merge this - the ecp errors seem unrelated. Thanks!

**Comment by danlipsa** [comment]
@burlen Ping.

**Comment by burlen** [comment]
@danlipsa OK, merging. added you to the team, in the future you can do it on your own

**Comment by danlipsa** [comment]
Great. Thanks!

**Pull request merged**